extensions/git/src/commands.ts

```diff
@@ -3658,6 +3665,12 @@ export class CommandCenter {
 					case GitErrorCodes.PushRejected:
 						message = l10n.t('Can\'t push refs to remote. Try running "Pull" first to integrate your changes.');
 						break;
+					case GitErrorCodes.ForcePushRejected:
+						message = l10n.t('Can\'t force push refs to remote. Check if force push is not prohibited by remote.');
+						break;
+					case GitErrorCodes.SaferForcePushRejected:
+						message = l10n.t('Can\'t force push refs to remote. Check if force push is not prohibited by remote or rebase to the latest remote branch first.');
```

Comment:
Should the second sentence be "Check if force push is not prohibited by the remote or pull the latest changes from the remote branch first."

Reply:
I'll fully comply with your suggestion.

Reply:
@lszomoru Could you tell me why did you tell me to change "`rebase to`" to "`pull`"?

Reply:
@lszomoru Is this just because Git itself makes such suggestion? Well, in that case, it's no wonder...